plugins/azad-proxy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: azad-proxy
+spec:
+  platforms:
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-darwin-amd64.zip
+    sha256: 51ae0098cbbb0b5da01f5b086c02474a8b74012e5f373c2dbeea36ad659c65e2
+    bin: kubectl-azad_proxy
+    files:
+    - from: kubectl-azad_proxy
```

Comment:
jfyi, no need to change: The binary name can be anything here---krew users will always get the plugin executable as given by the `metadata.name` field.

Reply:
@corneliusweig ah, good to know! I'm compiling it like this and hopefully the only way people are going to use it is through Krew :)